[2024-06-19T08:22:25.826+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T08:22:25.890+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-06-19T08:22:16.128997+00:00 [queued]>
[2024-06-19T08:22:25.907+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-06-19T08:22:16.128997+00:00 [queued]>
[2024-06-19T08:22:25.908+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-19T08:22:25.967+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_wikipedia_data> on 2024-06-19 08:22:16.128997+00:00
[2024-06-19T08:22:25.985+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_flow', 'transform_wikipedia_data', 'manual__2024-06-19T08:22:16.128997+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia_flow.py', '--cfg-path', '/tmp/tmpa3i02l7w']
[2024-06-19T08:22:25.992+0000] {standard_task_runner.py:91} INFO - Job 22: Subtask transform_wikipedia_data
[2024-06-19T08:22:26.001+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2928) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-19T08:22:26.003+0000] {standard_task_runner.py:63} INFO - Started process 2930 to run task
[2024-06-19T08:22:26.167+0000] {task_command.py:426} INFO - Running <TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-06-19T08:22:16.128997+00:00 [running]> on host dd278de8866b
[2024-06-19T08:22:26.593+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ayush Goswami' AIRFLOW_CTX_DAG_ID='wikipedia_flow' AIRFLOW_CTX_TASK_ID='transform_wikipedia_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T08:22:16.128997+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-19T08:22:16.128997+00:00'
[2024-06-19T08:22:26.596+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T08:22:29.990+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f89cc9bdfd0>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /search?q=Rungrado+1st+of+May+Stadium+%2C%C2%A0North+Korea&format=json&limit=1
[2024-06-19T08:25:04.170+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=RheinEnergieSTADION%2C%C2%A0Germany%0A&format=json&limit=1
[2024-06-19T08:25:45.546+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Stade+Mohammed+V+%2C%C2%A0Morocco&format=json&limit=1
[2024-06-19T08:26:31.025+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Estadio+Jos%C3%A9+Pachencho+Romero%2C%C2%A0Venezuela&format=json&limit=1
[2024-06-19T08:27:20.795+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/pipeline/wikipedia_pipeline.py:76 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-06-19T08:27:20.866+0000] {python.py:237} INFO - Done. Returned value was: ok
[2024-06-19T08:27:20.867+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T08:27:20.957+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=wikipedia_flow, task_id=transform_wikipedia_data, run_id=manual__2024-06-19T08:22:16.128997+00:00, execution_date=20240619T082216, start_date=20240619T082225, end_date=20240619T082720
[2024-06-19T08:27:21.032+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T08:27:21.066+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T08:27:21.090+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
